packages/connect-web/package.json

```diff
@@ -23,6 +28,9 @@
       "require": "./dist/cjs/index.js"
     }
   },
+  "devDependencies": {
+    "webdriverio": "^8.36.1"
+  },
```

Comment:
This package uses @connectrpc/connect-conformance, right? It needs a dev dependency on it.

Same for all other packages using @connectrpc/connect-conformance.